dashboard/client/src/App.tsx

```diff
@@ -229,7 +231,8 @@ const App = () => {
                       />
                     </Route>
                   </Route>
-                  <Route element={<Actors />} path="actors" />
+                  <Route element={<Actors newIA />} path="actors" />
+                  <Route element={<ActorDetailPage />} path="actors/:id" />
```

Comment:
This should be nested under Actors similar to how the Jobs and Clusters page works